src/Explorer/Menus/NotificationConsole/NotificationConsoleComponent.tsx

```diff
@@ -308,7 +308,8 @@ const PrPreview = (props: { pr: string }) => {
 export const NotificationConsole: React.FC = () => {
   const setIsExpanded = useNotificationConsole((state) => state.setIsExpanded);
   const isExpanded = useNotificationConsole((state) => state.isExpanded);
-  const consoleData = useNotificationConsole((state) => state.consoleData);
+  //eslint-disable-next-line
```

Comment:
Tried using **Optional Chaining operator** however it was causing the typescript error. While using NOT operator typescript error is getting resolved with eslint warning. Hence eslint disable next line attribute is used here.